### PR TITLE
Ruby/Rails Path: Introduction && Basic Ruby sections: Rewrite headings following the Style Guide

### PR DESCRIPTION
**Pull request by headlessNode**
## Because

- Change the headings according to Style Guide requirement.

## This PR

- Change the headings of the Ruby introduction section to sentence case.
- Change the headings of the Ruby basic ruby section to sentence case.

## Issue

- Related to #26169

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
That's quick! I'll review it right away. I'll also change the "related to" into the correct one for I've missed that in updating the text :)

**Review by ManonLef — CHANGES_REQUESTED** [review]
Thanks so much for doing this! The `Loops` lesson is the only one I have left a comment on since it seems to have a few more headers that need updating.

File: ruby/basic_ruby/loops.md
[the remark this answers is not in the document]
Comment (headlessNode) [reply ruby/basic_ruby/loops.md:1]: Hi, I just updated the loops.md please check.

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Thank you again! Much appreciated

**Pull request merged**